src/index.css

```diff
@@ -1,3 +1,4 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap');
```

Comment:
oh i didnt know we can import google fonts in CSS too!

Reply:
yup:)

Reply:
if inter is already imported as default, then i think we don't need to import this

Reply:
done